packages/hash/frontend/src/blocks/page/BlockContextMenu/LoadEntityMenuContent.tsx

```diff
@@ -33,16 +34,19 @@ export const LoadEntityMenuContent: VFC<LoadEntityMenuContentProps> = ({
   // and this can cause some bugs.
   // @todo make this a subscription
   const entityStore = entityStorePluginState(blockView.editorView.state).store;
-  const blockData = entityId ? entityStore.saved[entityId] : null;
+  // savedEntity and blockEntity are the same. savedEntity variable
+  // is needed to get proper typing on blockEntity
+  const savedEntity = entityId ? entityStore.saved[entityId] : null;
+  const blockEntity = isBlockEntity(savedEntity) ? savedEntity : null;
   const searchInputRef = useRef<HTMLInputElement>(null);
   const { data: entities, loading } = useAccountEntities({
     accountId,
     entityTypeFilter: {
-      componentId: isBlockEntity(blockData)
-        ? blockData?.properties.componentId
+      componentId: isBlockEntity(blockEntity)
+        ? blockEntity?.properties.componentId
         : undefined,
     },
-    skip: !(isBlockEntity(blockData) && !!accountId),
+    skip: !(isBlockEntity(blockEntity) && !!accountId),
```

Comment:
You've just checked `blockEntity` is of type `BlockEntity`, or its `null` - so you don't need to check it again. It's type is currently `BlockEntity | null` - so you can just check it isn't null